zerver/models.py

```diff
@@ -220,6 +220,28 @@ def get_display_recipient(recipient: "Recipient") -> DisplayRecipientT:
     )
 
 
+def get_recipient_ids(
+    recipient: Optional["Recipient"], user_profile_id: int
+) -> Tuple[List[int], str]:
+    if recipient is None:
+        _type = ""
```

Comment:
We should come up with a better variable name for this; maybe `recipient_type_str` is fine.